src/cartservice/tests/cartservice.tests.csproj

```diff
@@ -7,9 +7,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.Net.Client" Version="2.36.0" />
-    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="5.0.4" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+    <PackageReference Include="Grpc.Net.Client" Version="2.40.0" />
+    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="5.0.11" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
```

Comment:
I was slightly worried about this major version bump of `Microsoft.NET.Test.Sdk`, but it looks like the `CartServiceTests.cs` unit tests are passing in the GitHub checks:
`Passed!  - Failed:     0, Passed:     3, Skipped:     0, Total:     3, ... cartservice/tests/bin/Debug/net5.0/cartservice.tests.dll (net5.0)`